app/resources/views/pages/edits.txt

```diff
@@ -0,0 +1,5 @@
+
+	
+		if ( isProfileChanged() ) {
+		return '';
```

Comment:
Fix indentation here.  Maybe you have a mix of tabs and regular spaces.  Try to use the same whitespace characters for indentation that the rest of the file uses.

Make it so github's comparison here shows it looking good.